src/health-checks/health.controller.ts

```diff
@@ -17,25 +16,24 @@ export class HealthController {
                 health: await this.isHealthy(),
                 version: Environments.getVersion(),
             };
-        } catch (e) {
+        } catch (e: any) {
             this.logger.error(`Health failed. Original message: "${e.message}".`);
             throw new HttpException(e.message, HttpStatus.SERVICE_UNAVAILABLE);
         }
     }
 
-    private async isHealthy (): Promise<object> {
-        return new Promise<object>(async (resolve: Function, reject: Function): Promise<void> => {
-            try {
-                const responses: AxiosResponse<string>[] = await this.pingServices(servicesToPing);
-                resolve({
-                    databases: [],
-                    services: responses.map((response: AxiosResponse<string>): string =>
-                        `${response.request.res.responseUrl} ${response.data}`),
-                });
-            } catch (e) {
-                reject(e);
-            }
-        });
+    private async isHealthy (): Promise<any> {
+        try {
+            const responses: AxiosResponse<string>[] = await this.pingServices(servicesToPing);
+
+            return Promise.resolve({
+                databases: [],
+                services: responses.map((response: AxiosResponse<string>): string =>
+                    `${response.request.res.responseUrl} ${response.data}`),
+            });
+        } catch (e) {
+            return Promise.reject(e);
```

Comment:
catching and rejecting is the same as not catching at all no? or you could re-throw an actual error